### ISSUE TITLE
error: failed to run custom build command for `ffmpeg-sys-next v5.1.1` -> Solution: Stick to LLVM 15 for build

### ISSUE THREAD

**Issue by thebinarymutant** [spacedriveapp/spacedrive#677]
### Describe the bug

Running `pnpm desktop dev` fails to compile `ffmpeg-sys-next = "5.1.1"` 

Sharing my stack trace via [pastebin](https://pastebin.com/WWKPUzXk) to make it more clear as GitHub issues has a character limit.

### Platform and versions

```Shell
❯ pnpm --version && cargo --version && rustc --version
7.30.5
cargo 1.68.2 (6feb7c9cf 2023-03-26)
rustc 1.68.2 (9eb3afe9e 2023-03-27)

❯ uname -a
Darwin MacBook-Pro.local 22.3.0 Darwin Kernel Version 22.3.0; root:xnu-8792.81.3~2/RELEASE_X86_64 x86_64
```

### Stack trace

https://pastebin.com/WWKPUzXk

**Comment by HeavenVolkoff** [comment]
I had a very similar problem when compiling on Windows recently. The culprit was a new major update to LLVM (Version 16), which is breaking the `ffmpeg-sys-next` `build.rs` script. I reverted to LLVM 15.0.7 and was able to compile successfully. You can check what version you have with clang --version. If you have version 16, try reverting to version 15 and then compiling again.

**Comment by thebinarymutant** [comment]
@HeavenVolkoff Thank you, it worked after reverting to `LLVM 15.0.7`,

TLDR;
Stick to LLVM 15 for builds
